### ISSUE TITLE
Please consider adding U+2AA0A 𪨊 and U+20BBE 𠮾

### ISSUE THREAD

**Issue by Synacrystal** [adobe-fonts/source-han-sans#240]
“𪨊” is the simplification of “㞞”, and is widely used in spoken Chinese (people usually replace it with “怂” while typing). “𠮾” is an interjection similar to “嗯” and can be seen in early novels.

**Comment by LIXiangChen** [comment]
Agree, the publishing/broadcasting industry requires that use "𪨊" instead of its common word (ie typo) "怂".

**Comment by muellert** [comment]
When I recently looked at the Unihan dictionary, I've noticed that there may be a lot of glyphs missing.

![Image](https://github.com/user-attachments/assets/44c1dead-014b-4936-988a-cd963534bf14)

It would be great if at least one font could be made complete, so we can see a glyph instead of a box with six digits inside.
